packages/babel-preset-react-app/dev.js

```diff
@@ -9,5 +9,5 @@
 const create = require('./create');
 
 module.exports = function(api, opts) {
-  return create(api, opts, 'development');
+  return create(api, { helpers: false, ...(opts || {}) }, 'development');
```

Comment:
It's fine to spread undefined. I don't think you need `|| {}`

Reply:
This is object spread though. Is it supported in all envs we target?

Reply:
look at that. they thought of everything.